package.json

```diff
@@ -38,7 +38,7 @@
     "chalk": "^4.0.0",
     "clean-webpack-plugin": "^3.0.0",
     "cli-table": "0.3.1",
-    "compound-components": "https://github.com/compound-finance/compound-components.git#e1734102d56ffa14727f10435c68288d9f058a6d",
+    "compound-components": "https://github.com/compound-finance/compound-components.git#7d8959111595c0ac4ab97cff648fef18ee41d50a",
```

Comment:
once we merge the compound components then we'd better update to the main branch sha no?

Reply:
Yup, will update sha after components, merged to main